packages/x-charts/src/LineChart/AreaPlot.tsx

```diff
@@ -18,7 +18,74 @@ export interface AreaPlotSlotProps {
 
 export interface AreaPlotProps
   extends React.SVGAttributes<SVGSVGElement>,
-    Pick<AreaElementProps, 'slots' | 'slotProps'> {}
+    Pick<AreaElementProps, 'slots' | 'slotProps' | 'skipAnimation'> {}
+
+const useCompletedData = () => {
+  const seriesData = React.useContext(SeriesContext).line;
+  const axisData = React.useContext(CartesianContext);
+
+  if (seriesData === undefined) {
+    return [];
+  }
+
+  const { series, stackingGroups } = seriesData;
+  const { xAxis, yAxis, xAxisIds, yAxisIds } = axisData;
+  const defaultXAxisId = xAxisIds[0];
+  const defaultYAxisId = yAxisIds[0];
+
+  return stackingGroups.flatMap(({ ids: groupIds }) => {
+    return groupIds.flatMap((seriesId) => {
+      const {
+        xAxisKey = defaultXAxisId,
+        yAxisKey = defaultYAxisId,
+        stackedData,
+        data,
+        connectNulls,
+      } = series[seriesId];
+
+      const xScale = getValueToPositionMapper(xAxis[xAxisKey].scale);
+      const yScale = yAxis[yAxisKey].scale;
+      const xData = xAxis[xAxisKey].data;
+
+      if (process.env.NODE_ENV !== 'production') {
+        if (xData === undefined) {
+          throw new Error(
+            `MUI X Charts: ${
```

Comment:
Nitpick: Currently there are 3 instances where the error is prefixed with `MUI X` in the Charts package instead of `MUI X Charts` like in other cases.
Is there a reason for it?
WDYT about aligning those? 🤔